### PR TITLE
Javascript/javascript1/week1

### PR DESCRIPTION
**Pull request by Poorooshka**

### CODE REVIEW

File: javascript/javascript1/week1/Future_Age.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <title>A future age calculator</title>
+    <meta name="viewport" charset="utf-8" width="device-width, initial-scale=1.0">
+
+</head>
+
+<body>
+
+
+    <script>
+        const yearOfBirth = 1984;
```
Comment (aina21) [inline javascript/javascript1/week1/Future_Age.html]: add a script to file

File: javascript/javascript1/week1/Future_Age.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        let yearFuture = 2045;
```
Comment (aina21) [inline javascript/javascript1/week1/Future_Age.html]: use const

File: javascript/javascript1/week1/Future_Age.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+        let age = yearFuture - yearOfBirth;
```
Comment (aina21) [inline javascript/javascript1/week1/Future_Age.html]: always use const until it doesn't let you

File: javascript/javascript1/week1/House_Price.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <body>
+
+        <script>
+
+
+            function evaluateHousePrice (name, width, height, depth, gardenSize, houseCost) 
+            {
+
+
+                volumeInMeters = width*height*depth;
```
Comment (aina21) [inline javascript/javascript1/week1/House_Price.html:16]: you need to write const or let before a variable to have function scope, now volumemeters automatically becomes a global variable.
you can read more about it 
https://www.programiz.com/javascript/variable-scope

**Review by aina21 — CHANGES_REQUESTED** [review]
good job 👍

File: javascript/javascript1/week1/Future_Age.html (hunk 1 above)
Comment (Poorooshka) [reply javascript/javascript1/week1/Future_Age.html]: Hi, thanks a lot. Do you mean a separate js file?

Comment (aina21) [reply javascript/javascript1/week1/Future_Age.html]: yes

**Pull request merged**